library/couchdb

```diff
@@ -1,14 +1,17 @@
-# maintainer: Joan Touzet <wohali@apache.org> (@wohali)
+Maintainers: Joan Touzet <wohali@apache.org> (@wohali)
+GitRepo: https://github.com/apache/couchdb-docker.git
 
-latest: git://github.com/apache/couchdb-docker@1a7c4254c158a194ff195da6ebfed910d24a95b7 2.1.1
-2.1.1: git://github.com/apache/couchdb-docker@1a7c4254c158a194ff195da6ebfed910d24a95b7 2.1.1
-2.1: git://github.com/apache/couchdb-docker@1a7c4254c158a194ff195da6ebfed910d24a95b7 2.1.1
-2: git://github.com/apache/couchdb-docker@1a7c4254c158a194ff195da6ebfed910d24a95b7 2.1.1
+Tags: latest, 2.1.1, 2.1, 2
```

Comment:
It would be prudent to move `latest` to the end of this list instead of the beginning, but I do realize you've simply translated the existing file into the new format as-is.

See https://github.com/docker-library/official-images/issues/3467 for an example of the sort of issue caused by this ordering (tldr: our tooling assumes "most specific" tag comes first).